tools/purge_duplicate_rubyrep_triggers.rb

```diff
@@ -16,6 +16,9 @@
 GROUP BY relname
 SQL
 
+REGION = File.exists?("./REGION") ? File.read("./REGION").strip : "0"
```

Comment:
Is it possible to not use the `REGION` file to discover this logic?

@carbonin outside of vmdb, do you know of a function similar to `MiqRegion.my_region_number`?